src/coreComponents/mesh/generators/VTKMeshGenerator.cpp

```diff
@@ -998,6 +1004,8 @@ real64 VTKMeshGenerator::writeNodes( CellBlockManager & cellBlockManager ) const
  * @param[in] regionsTetra map from region index to the tetra indexes in this region
  * @param[in] regionsWedges map from region index to the wedges indexes in this region
  * @param[in] regionsPyramids map from region index to the pyramids indexes in this region
+ * @param[in] regionsPentagonalPrisms map from region index to the pentagonal prisms indexes in this region
+ * @param[in] regionsHexagonalPrisms map from region index to the hexagonal prisms indexes in this region
```

Comment:
Hmm, this `doxygen` clearly looks out of sync?

Reply:
Done